pkg/service/restore/progress.go

```diff
@@ -0,0 +1,205 @@
+// Copyright (C) 2023 ScyllaDB
+
+package restore
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/scylladb/gocqlx/v2"
+	"github.com/scylladb/gocqlx/v2/qb"
+	"github.com/scylladb/scylla-manager/v3/pkg/schema/table"
+	"github.com/scylladb/scylla-manager/v3/pkg/scyllaclient"
+	"github.com/scylladb/scylla-manager/v3/pkg/util/uuid"
+)
+
+var (
+	zeroTime time.Time
+	maxTime  = time.Unix(1<<62-1, 0).UTC()
```

Comment:
nit (completely unimportant): this is not the golang maxTime.

https://stackoverflow.com/questions/25065055/what-is-the-maximum-time-time-in-go

it doesn't matter for us ofc.